ReSharper.FSharp/test/data/features/daemon/identifierHighlighting/Type aliases 03 - Units of measure.fs

```diff
@@ -0,0 +1,21 @@
+[<Measure>] type m
+[<Measure>] type s
+
+type MeasureBasicType = int<m/s>
+
+type MeasureRecord<[<Measure>] 'u> = { x : float<'u>; y : float<'u>; z : float<'u> }
```

Comment:
The same for the second and third fields.

Reply:
Simplified tests. Fixed some bugs since then.